gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceGuiceModule.java

```diff
@@ -188,6 +189,9 @@ public void configure(Binder binder) {
       binder.bind(FlowExecutionResourceHandler.class).to(GobblinServiceFlowExecutionResourceHandler.class);
     }
 
+    // TODO: may need to move this initialization higher
```

Comment:
this is just binding configuration. order does not matter here.

Reply:
i mean, object initialization happens in the order they are required, not in the order their binding is defined in this method.